circle.yml

```diff
@@ -1,13 +1,19 @@
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I've no idea really about the default behaviour, I just copied this from a Gist I found. I'll leave `-o` out and see if it still works.